docs/modules/ROOT/pages/tasks/build.adoc

```diff
@@ -0,0 +1,133 @@
+
+[[apply-tags-0.1]]apply-tags (0.1):: Applies additional tags to the built image. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/apply-tags/0.1/README.md[README.md]
+
+[[buildah-0.1]]buildah (0.1):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah/0.1/README.md[README.md]
+
+[[buildah-0.2]]buildah (0.2):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah/0.2/README.md[README.md] https://github.com/konflux-ci/build-definitions/tree/main/task/buildah/0.2/MIGRATION.md[MIGRATION.md]
+
+[[buildah-oci-ta-0.1]]buildah-oci-ta (0.1):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-oci-ta/0.1/README.md[README.md]
+
+[[buildah-oci-ta-0.2]]buildah-oci-ta (0.2):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-oci-ta/0.2/README.md[README.md] https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-oci-ta/0.2/MIGRATION.md[MIGRATION.md]
+
+[[buildah-remote-0.1]]buildah-remote (0.1):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-remote/0.1/README.md[README.md]
+
+[[buildah-remote-0.2]]buildah-remote (0.2):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-remote/0.2/README.md[README.md] https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-remote/0.2/MIGRATION.md[MIGRATION.md]
+
+[[buildah-remote-oci-ta-0.1]]buildah-remote-oci-ta (0.1):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also:
+
+[[buildah-remote-oci-ta-0.2]]buildah-remote-oci-ta (0.2):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also: https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-remote-oci-ta/0.2/README.md[README.md] https://github.com/konflux-ci/build-definitions/tree/main/task/buildah-remote-oci-ta/0.2/MIGRATION.md[MIGRATION.md]
+
+[[buildah-rhtap-0.1]]buildah-rhtap (0.1):: Buildah task builds source code into a container image and pushes the image into container registry using buildah tool. See also:
```

Comment:
Did we decide that we will not filter the list of tasks to remove ones like this?

Reply:
Good catch! Let's filter it.